### PR TITLE
Upgrade to 11.0

### PR DESCRIPTION
**Pull request by timunie**
need to edit ReadMe as well

### CODE REVIEW

File: Avalonia.MusicStore/Roots.xml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 ﻿<linker>
```
Comment (maxkatz6) [inline Avalonia.MusicStore/Roots.xml:1]: This file should be removed, if we use compiled bindings everywhere. Just need to double check with AOT

File: Avalonia.MusicStore/ViewLocator.cs [hunk 2]
```diff
@@ -7,9 +7,9 @@ namespace Avalonia.MusicStore
 {
     public class ViewLocator : IDataTemplate
     {
-        public IControl Build(object data)
+        public Control Build(object? data)
         {
-            var name = data.GetType().FullName!.Replace("ViewModel", "View");
+            var name = data!.GetType().FullName!.Replace("ViewModel", "View");
```
Comment (maxkatz6) [inline Avalonia.MusicStore/ViewLocator.cs:12]: Ah, nevermind, if we use reflection here, we still need that Roots.xml file. 
I guess we shouldn't remove this view locator, if it's used in the tutorials, without updating tutorials first.

**Review by maxkatz6 — APPROVED** [review]

File: Avalonia.MusicStore/ViewLocator.cs (hunk 2 above)
Comment (timunie) [reply Avalonia.MusicStore/ViewLocator.cs:12]: Yeah in long term I want to replace MusicStore by somehting more useful.

File: Avalonia.MusicStore/Roots.xml (hunk 1 above)
Comment (timunie) [reply Avalonia.MusicStore/Roots.xml:1]: ViewLocator ;-)

**Pull request merged**